bridge/engine/EngineMixer.cpp

```diff
@@ -1019,6 +1019,7 @@ void EngineMixer::runDominantSpeakerCheck(const uint64_t engineIterationStartTim
     bool dominantSpeakerChanged = false;
     bool videoMapChanged = false;
     bool audioMapChanged = false;
```

Comment:
I think we can remove this flag altogether. We need to keep if videomapping changed as that is only reason to send UMM to clients.

Reply:
But what about UMM to barbell? W/o UMM after one endpoint has replaced another in the audio ssrc map, receiver will be confused and received audio will be attributed to the wrong endpoint, which will led to wrong decision on the active media list.